WordPress/Classes/Extensions/UIImage+Exporters.swift

```diff
@@ -112,6 +112,16 @@ extension UIImage: ExportableAsset
         }
     }
 
+    func exportOriginalImage(toURL: NSURL, successHandler: SuccessHandler, errorHandler: ErrorHandler)
+    {
+        do {
+            try self.writeToURL(toURL, type:originalUTI()!, compressionQuality:0.9, metadata: nil)
```

Comment:
I'm guessing this `compressionQuality` isn't actually computed since we're writing a .GIF, but just in case, should this instead default to `1.0` for a .GIF?